tests/hotplug/rolloutstrategy.go

```diff
@@ -39,17 +37,19 @@ var _ = Describe("[sig-compute]VM Rollout Strategy", decorators.SigCompute, Seri
 
 	Context("When using the Stage rollout strategy", func() {
 		BeforeEach(func() {
+			virtClient = kubevirt.Client()
 			rolloutStrategy := pointer.P(v1.VMRolloutStrategyStage)
-			rolloutData, err := json.Marshal(rolloutStrategy)
-			Expect(err).To(Not(HaveOccurred()))
-
-			data := fmt.Sprintf(`[{"op": "replace", "path": "/spec/configuration/vmRolloutStrategy", "value": %s}]`, string(rolloutData))
+			err := config.RegisterKubevirtConfigChange(
+				config.WithVMRolloutStrategy(rolloutStrategy),
+			)
+			Expect(err).ToNot(HaveOccurred())
 
-			kv := libkubevirt.GetCurrentKv(virtClient)
-			Eventually(func() error {
-				_, err := virtClient.KubeVirt(flags.KubeVirtInstallNamespace).Patch(context.Background(), kv.Name, types.JSONPatchType, []byte(data), metav1.PatchOptions{})
-				return err
-			}, 10*time.Second, 1*time.Second).ShouldNot(HaveOccurred())
+			currentKv := libkubevirt.GetCurrentKv(virtClient)
+			config.WaitForConfigToBePropagatedToComponent(
+				"kubevirt.io=virt-controller",
+				currentKv.ResourceVersion,
+				config.ExpectResourceVersionToBeLessEqualThanConfigVersion,
+				time.Minute)
```

Comment:
I don't completely follow this change, is it possible to introduce it in a dedicate commit?